lib/flutter/flutter_tools.dart

```diff
@@ -40,6 +43,14 @@ class FlutterToolsManager implements Disposable {
         'flutter:upgrade',
         _upgrade
       ));
+      disposables.add(atom.commands.add(
+        'atom-workspace',
+        'flutter:connect',
```

Comment:
The command name will show up in the command palette dialog as `Flutter: Connect`. We should give it a name that indicates a bit more what it's for - something like `flutter:connect-remote-debugger`.